homeassistant/components/awair/config_flow.py

```diff
@@ -0,0 +1,76 @@
+"""Config flow for Awair."""
+
+from typing import Union
+
+from python_awair import Awair
+from python_awair.exceptions import AuthError, AwairError
+import voluptuous as vol
+
+from homeassistant.config_entries import CONN_CLASS_CLOUD_POLL, ConfigFlow
+from homeassistant.const import CONF_ACCESS_TOKEN
+from homeassistant.helpers.aiohttp_client import async_get_clientsession
+
+from .const import DOMAIN, LOGGER  # pylint: disable=unused-import
+
+
+class AwairFlowHandler(ConfigFlow, domain=DOMAIN):
+    """Config flow for Awair."""
+
+    VERSION = 1
+    CONNECTION_CLASS = CONN_CLASS_CLOUD_POLL
+
+    def __init__(self) -> None:
+        """Set up the Awair configuration flow."""
+        self._errors = {}
```

Comment:
Form errors should not be an instance attribute as they are local to each step. Please create local variables for errors instead, one per step that needs to show a form.

Reply:
Sure - for what it's worth, this kind of stuff is not incredibly clear from documentation, and I cargo-culted this from elsewhere within the codebase. 😄  I'll update later tonight.